dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/ExperimentFactory.java

```diff
@@ -0,0 +1,233 @@
+package com.dotcms.rest.api.v1.experiment;
+
+import static com.dotcms.util.CollectionsUtils.map;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.rest.api.v1.experiment.Experiment.ExperimentStatus;
+import com.dotcms.util.JsonUtil;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.htmlpageasset.model.IHTMLPage;
+import com.dotmarketing.portlets.rules.model.Rule;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+
+public class ExperimentFactory {
```

Comment:
Why a factory does not follow the Factory patterns of DOTCMS, 
1. Should implement an interface and be provided by the FactoryLocator
2. If needed provides the cache layer
3. Do not use static method in a favor of instance methods

Reply:
it is really a POC, really I am not thinking in this PR to be merge